hw/ip_templates/racl_ctrl/doc/interfaces.md.tpl

```diff
@@ -0,0 +1,4 @@
+${"#"} Hardware Interfaces
+
+<!-- BEGIN CMDGEN util/regtool.py --interfaces ./hw/top_${topname}/ip_autogen/racl_ctrl/data/racl_ctrl.hjson -->
```

Comment:
Are there strange characters before "interfaces"? (Github's diff view shows them in red)

Reply:
It seems it gets confused with being a diff control statement. Nothing in the source file.